### PR TITLE
Knowledge: Sensitive data tagging

### PR DESCRIPTION
**Pull request by nishithsharma9**
Nishith Sharma <ns5014@nyu.edu>

**Describe the contribution to the taxonomy**

- To analyze and classify various data elements across different domains (e.g., mortgage, clinical
trials, product domain) to determine if they are considered Sensitive Data Elements?
- Each classification should be accompanied by a reasoning that explains why the data element is or is not
considered Sensitive Data.

**Input given at the prompt**
```
Is "ModelNum" considered sensitive data in the product domain?
```

**Response from the original model**
```
Yes, "ModelNum" is considered personally sensitive data in the product domain.
```

**Response from the fine-tuned model**
```
No, "ModelNum" is not sensitive data as it refers to the model number of a product, not associated with personal data, in the product domain.
```

**Contribution checklist**

- [x] The contribution was tested with `ilab generate`
- [x] No errors or warnings were produced by `ilab generate`
- [x] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [x] The `qna.yaml` file contains at least 5 `seed_examples`
- [x] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [x] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [x] Content does not include PII or otherwise sensitive or confidential information
- [x] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

File: knowledge/legal/compliance/sensitive_data_tagging/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+task_description: "To analyze and classify various data elements across different domains (e.g., mortgage, clinical
+trials, product domain) to determine if they are considered Sensitive Data Element (SDE). Each
+classification should be accompanied by a reasoning that explains why the data element is or is not
+considered SDE."
+created_by: Nishith Sharma
```
Comment (jjasghar) [inline knowledge/legal/compliance/sensitive_data_tagging/qna.yaml:5]: Please put your GitHub Username here.

File: knowledge/legal/compliance/sensitive_data_tagging/qna.yaml
Comment (jjasghar) [inline knowledge/legal/compliance/sensitive_data_tagging/qna.yaml:1]: We also need an `attribution.txt` file, please check the Knowledge guide for the format.

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please take a moment to make these fixes and also resolve the DCO.

**Comment by nishithsharma9** [comment]
Tagging here for reference: @adampingel

**Pull request closed without merging**